infra/main.tf

```diff
@@ -102,3 +102,58 @@ resource "aws_cloudwatch_event_target" "cloudwatch_event_target" {
     "table" : aws_dynamodb_table.dynamodb_table.name
   })
 }
+
+resource "aws_elasticsearch_domain" "elasticsearch_domain" {
```

Comment:
Not sure if it is needed - so it is more of a question - DO we need to also add this resource to the iam role attachment? (resource is found at line 51, and my question is whether we need to add it to the services in line 52)

Reply:
Nice catch. It is required for a second lambda function. I'm working on a follow up PR which generates two IAM Roles. One for each lambda function.